tools/myst/enc/enc.c

```diff
@@ -521,6 +534,24 @@ static long _enter(void* arg_)
     /* Get the enclave size */
     enclave_size = __oe_get_enclave_size();
 
+    if (_get_config_id(config_id) != 0)
+    {
+        fprintf(stderr, "_get_config_id() failed\n");
+        assert(0);
+    }
+
+    myst_load_fssig(options->rootfs, &fssig);
+    if (_check_config_id(
+            config_id,
+            arg->config_data,
+            arg->config_size,
+            fssig.root_hash,
```

Comment:
With the future implementation of rootfs roothash info as json name/value pairs, and as part of the config_data passed in, the check will be to make sure sha256(config_data) = config_id. Then the code should check the iinfo in the rootfs roothash  name/value pairs matches `fssig.root_hash`.